tests/pytests/test_config.py

```diff
@@ -216,7 +216,7 @@ def testInitConfig():
 
     _test_config_str('GC_POLICY', 'fork')
     _test_config_str('GC_POLICY', 'default', 'fork')
-    _test_config_str('ON_TIMEOUT', 'return')
+    _test_config_str('ON_TIMEOUT', 'fail')
```

Comment:
Should be return?

Reply:
Don't think so, this is how it was before the change
It is testing that you can create an env with this config (not the default one)